docs/source/developer/contributing.rst

```diff
@@ -252,6 +252,25 @@ A code review should try to assess the following criteria:
 * docstrings, comments
 * test coverage
 
+
+Policy on large scale changes to code formatting
+''''''''''''''''''''''''''''''''''''''''''''''''
+
+Please note that pull requests making large scale changes to format the code
+base are in general not accepted. Such changes often increase the likelihood of
+merge conflicts for other pull requests, which inevitably take time and
+resources to resolve. They also require a lot of effort to check as Numba aims
+to compile code that is valid even if it is not ideal, i.e. testing
+``operator.eq``::
+
+    if x == None: # should be `if x is None:` but it's valid so don't change it!
```

Comment:
I think it's a little hard to understand this explanation as the explanation of the context is quite terse - perhaps the following would explain more clearly to a broader set of people:

```suggestion
to compile code that is valid even if it is not ideal. For example, in a test of
``operator.eq``::

    if x == None: # Valid code, even if the recommended form is `if x is None:`

This tests Numba's compilation of comparison with ``None``, and therefore
should not be changed, even though most style checkers will suggest it should.
```

What do you think?

Reply:
I think this makes it clearer, thanks for suggesting it.